src/coreclr/vm/comdelegate.cpp

```diff
@@ -2143,12 +2143,26 @@ extern "C" PCODE QCALLTYPE Delegate_GetMulticastInvokeSlow(MethodTable* pDelegat
         pCode->EmitLDC(0);
         pCode->EmitSTLOC(dwLoopCounterNum);
 
-        // Make the shape of the loop similar to what C# compiler emits
```

Comment:
Could you please also delete ` ILCodeLabel *checkCount = pCode->NewCodeLabel();` line above